lib/rules/prefer-spread.js

```diff
@@ -23,7 +23,8 @@ function isVariadicApplyCalling(node) {
         node.callee.property.name === "apply" &&
         node.callee.computed === false &&
         node.arguments.length === 2 &&
-        node.arguments[1].type !== "ArrayExpression"
+        node.arguments[1].type !== "ArrayExpression" &&
+        node.arguments[1].type !== "SpreadElement"
```

Comment:
Why was this condition added?
The reason of `node.arguments[1].type !== "ArrayExpression"` is that [no-useless-call](http://eslint.org/docs/rules/no-useless-call) rule warns it.